src/app/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+
+import { SoldProducts } from '../product';
+import { ProductService } from '../product.service';
+
+@Component({
+  selector: 'app-progress-bar',
+  templateUrl: './progress-bar.component.html',
+  styleUrls: ['./progress-bar.component.scss']
+})
+export class ProgressBarComponent implements OnInit {
+  progressValue: number = 0;
```

Comment:
This should be @Input. Progress bar component should be dummy component

Reply:
Done :)